ui/apps/platform/src/Containers/SystemHealth/Components/DiagnosticBundleDialogBox.tsx

```diff
@@ -117,9 +120,30 @@ const DiagnosticBundleDialogBox = (): ReactElement => {
             startingTimeObject,
             isStartingTimeValid,
         });
-        downloadDiagnostics(queryString).finally(() => {
-            setIsDownloading(false);
-        });
+        downloadDiagnostics(queryString)
+            .then(() => {
+                setAlertDownload(null);
+            })
+            .catch((error) => {
+                setAlertDownload(
+                    <Alert
+                        title="Downloading diagnostic bundle failed."
+                        variant={AlertVariant.danger}
+                        isInline
+                    >
+                        <p>
+                            <b>{error.message}</b>
+                            <br />
+                            If timeout is exceeded, repeat request with increased timeout using the
```

Comment:
My first impression of **repeat request** was:
1. use roxctl to increase the timeout
2. repeat the request via user interface

My second impression is if timeout is exceeded when customer attempts to download via user interface, then use roxctl command line **instead** of user interface.

Can we rewrite the sentence to remove the ambiguity?